shared/cloud/ess-getting-started.asciidoc

```diff
@@ -13,9 +13,10 @@ There's no faster way to get started than with our hosted {ess} on Elastic Cloud
 
 . Click *Create deployment*.
 
-. Select a solution and give your deployment a name.
+. Give your deployment a name.
 
 . Click *Create deployment* and download the password for the `elastic` user.
 // end::generic[]
 
-That’s it! Now that you are up and running, it’s time to get some data into {kib}. Click *Launch Kibana*.
+That’s it! Now that you are up and running, it’s time to get some data into
+{kib}. You will automatically be launched into {kib}.
```

Comment:
```suggestion
{kib}. {kib} will open as soon as your deployment is ready.
```
It's not as fun, but "you will automatically be launched into" suggests to me that the reader will be blasted off into ~~orbit~~ Kibana. :-)